pkg/sql/create_table.go

```diff
@@ -1372,10 +1372,11 @@ func MakeTableDesc(
 			}
 		case *tree.UniqueConstraintTableDef:
 			idx := sqlbase.IndexDescriptor{
-				Name:             string(d.Name),
-				Unique:           true,
-				StoreColumnNames: d.Storing.ToStrings(),
-				Version:          indexEncodingVersion,
+				Name:              string(d.Name),
+				Unique:            true,
+				StoreColumnNames:  d.Storing.ToStrings(),
+				Version:           indexEncodingVersion,
+				CreatedExplicitly: true,
```

Comment:
should we be setting this in `index_def` but not `constraint_elem` in sql.y instead?